db/data/regions/IT/en.yml

```diff
@@ -8,10 +8,12 @@ en:
             default: Street and house number
             optional: Street and house number (optional)
           errors:
-            blank: Enter a street and house number
+            blank_instructional: Enter a street and house number
+            blank_informative: Street and house number is missing
```

Comment:
Same comment as above.
If neither are present: Street and house number are missing
If one or the other is present: Street or house number is missing